server/app/services/program/predicate/PredicateValue.java

```diff
@@ -91,23 +102,35 @@ private static PredicateValue create(
    * @param question the question the predicate is applied to.
    */
   public String toDisplayString(QuestionDefinition question) {
-
     /* Special handling of "simple" question types, EG non-multivalued questions. */
 
     // Currency is stored as cents and displayed as dollars/cents with 2 cent digits.
     if (question.getQuestionType().equals(QuestionType.CURRENCY)) {
-      long storedCents = Long.parseLong(value());
-      long dollars = storedCents / 100;
-      long cents = storedCents % 100;
-      return String.format("$%d.%02d", dollars, cents);
+      if (type() == OperatorRightHandType.PAIR_OF_LONGS) {
+        return Splitter.on(", ")
```

Comment:
Yeah I thought about that but another layer of indirection seemed about even against the repetition.

Looking at it again, I decided to extract the splitting and trimming. Extracting the whole thing with a map function doesn't work as well because different branches use different collectors.

Open to naming suggestions on the helper... I went through:
* deserializeList
* stringListToStream
* listStringToStream
* parseListString

before landing on splitListString

Reply:
SGTM thank you!